SUMMARY.md

```diff
@@ -1,16 +1,16 @@
 # Summary
 
-- [Repo information](README.md)
+- [Intro](README.md)
 
 # Documentation
 
-- [Transactions overview](doc/README.md)
-- [Transactions in TiKV](doc/tikv/README.md)
+- [Transactions in TiKV](doc/tikv/transaction-in-tikv.md)
+- [Transactions handling newbiew perspective](doc/tikv/transaction-handling-newbie-perspective/transaction-handling-newbie-perspective.md)
 
 # Design docs
 
 - [Proposal for refactoring how transactional commands are scheduled and implemented](design/transaction-layer-refactoring.md)
-- [Async commit](design/async-commit/README.md)
+- [Async commit](design/async-commit/async-commit.md)
```

Comment:
Calling it `README.md` makes it render for `design/async-commit` directory.

Reply:
done

Reply:
done